Kiosk/Admin/AdminPanelViewController.swift

```diff
@@ -40,7 +39,8 @@ class AdminPanelViewController: UIViewController {
         if APIKeys.sharedKeys.stubResponses {
             auctionIDLabel.text = "STUBBING API RESPONSES\nNOT CONTACTING ARTSY API"
         } else {
-            let version = (Bundle.main.object(forInfoDictionaryKey: "CFBundleShortVersionString") as? String)  ?? "Unknown"
+            let version = (Bundle.main.object(forInfoDictionaryKey: "CFBundleShortVersionString")
+                as? String) ?? "Unknown"
```

Comment:
```swift
            let version = (Bundle.main.object(forInfoDictionaryKey: "CFBundleShortVersionString") as? String)  ?? "Unknown"
//-----------------------------------------------------------------------------|
            let version = (
                Bundle.main.object(
                    forInfoDictionaryKey: "CFBundleShortVersionString"
                ) as? String
            ) ?? "Unknown"
```

?

Reply:
I was using this version before but there were many situations in this PR where that looked odd. Prettier JS also never breaks after a grouping `(`, so I'm aligning with Prettier JS here.

---

See https://prettier.io/playground

```
(Bundle.main.object(fooFooFooFooFooFooFooFooFoo().fooFooFooFooFooFooFooFooFoo()) + true) * x
```

Reply:
Hmm, interesting. So the solution to avoid breaking on `as?` is to break the `Bundle.main...` line onto its own line?

Reply:
Yes. [Here's what happened](https://github.com/sirlantis/eidolon/pull/1/commits/d73e1cfd99b79c7c803bfa404576b28e82b6a7db#r159794630) before I started allowing breaking in front of `as` and preventing breaks in parenthesized expressions with just one element, which looked rather ugly.

Taking a closer look however, `as` as a much higher precedence than `??` (and `!=` in the linked example), so the parentheses are optional. So theoretically prettier could strip the parens and print (If I'd remove the "never break on a single element in a parenthesized group):

```
let version = Bundle.main.object(
        forInfoDictionaryKey: "CFBundleShortVersionString"
    ) as? String ?? "Unknown"
```

Would need to teach the formatter about precedence and associativity though so that it could do the stripping here.